src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,206 @@
+// @import "../../sass-utilities/all";
+
+.pf-c-inline-edit {
+  // Group
+  --pf-c-inline-edit__group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Value group
+  --pf-c-inline-edit__group--m-value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action group
+  --pf-c-inline-edit__group--m-action-group--item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight: 0;
+
+  // Valid action
+  --pf-c-inline-edit__action--m-valid--c-button--m-plain--Color: var(--pf-global--link--Color);
+  --pf-c-inline-edit__action--m-valid--c-button--m-plain--hover--Color: var(--pf-global--link--Color--hover);
+
+  // Icon group
+  --pf-c-inline-edit__group--m-icon-group--item--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Footer group
+  --pf-c-inline-edit__group--m-footer--MarginTop: var(--pf-global--spacer--xl);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-m-display-inline-flex {
+    display: inline-flex;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+// Group
+.pf-c-inline-edit__group {
+  display: flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__group--item--MarginRight);
+  }
+
+  // Value group
+  &.pf-m-value-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-value-group--item--MarginRight);
+  }
+
+
+  // Input group
+  &.pf-m-input-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--item--MarginRight);
+
+    .pf-m-action-group {
+      --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight);
+      --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+    }
+  }
+
+  // Action group
+  &.pf-m-action-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-action-group--item--MarginRight);
+  }
+
+  &.pf-m-input-group,
+  &.pf-m-action-group {
+    display: none;
+  }
+
+  // Icon group
+  &.pf-m-icon-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-icon-group--item--MarginRight);
+  }
+
+  // Inline group
+  &.pf-m-inline {
+    --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+  }
+
+  // Footer
+  &.pf-m-footer {
+    margin-top: var(--pf-c-inline-edit__group--m-footer--MarginTop);
+  }
+
+  // Column
+  &.pf-m-column {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+
```

Comment:
should you add a vertical spacer for the item(s) there? I think the same horizontal spacer size would work well.

Reply:
this is used to stack checkboxes and radios. it'll likely change in the future.

Reply:
adding top margin throws off layout